### PR TITLE
doc: add slides concerning secure storage, fix layout

### PR DESCRIPTION
**Pull request by gitfineon**
* Did some minor layout changes to the secure_storage.md file.
* Added more slides from linaro connect concerning this topic.

### CODE REVIEW

**Comment by jforissier** [comment]
Thanks @gitfineon. Please fix the `checkpatch` errors (https://travis-ci.org/OP-TEE/optee_os/builds/377948079#L2230-L2248), then:

`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jforissier** [comment]
@gitfineon looks good to me, there is one last step before I can merge this: could you please squash the two patches into one and add your `Signed-off-by: ...` and my `Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`, as described in https://github.com/OP-TEE/optee_os/blob/master/documentation/github.md#finalizing-your-contribution?

File: documentation/secure_storage.md [hunk 1]
```diff
@@ -33,57 +33,61 @@ The rest of this document describes the REE FS only.
...
-   If configured with CFG_RPMB_FS=y the protection against rollback is is
-   controlled by the TEE and is set to 1000. If CFG_RPMB_FS=n, there's no
-   protection against rollback, and the protection level is set to 0.
-
-### TEE File Structure In Linux File System
+Below is an excerpt from the specification, listing the most vital requirements:
+
+> 1. The Trusted Storage may be backed by non-secure resources as long as
+>    suitable cryptographic protection is applied, which MUST be as strong as
+>    the means used to protect the TEE code and data itself
```
Comment (etienne-lms) [inline documentation/secure_storage.md]: minor: add missing terminal dot.

File: documentation/secure_storage.md [hunk 2]
```diff
@@ -226,5 +231,8 @@ or crypto unit according to the method defined by your SoC vendor.
 
 ## Reference
 
-* [Secure Storage Presentation](http://www.slideshare.net/linaroorg/sfo15503-secure-storage-in-optee)
+* [SFO15 Secure Storage (slides)](http://connect.linaro.org/resource/sfo15/sfo15-503-secure-storage-in-op-tee/)
+* [LAS16 Secure Storage (slides)](http://connect.linaro.org/resource/las16/las16-504/)
+* [SFO17 Secure Storage (slides)](http://connect.linaro.org/resource/sfo17/sfo17-309/)
+* [HKG18 Secure Key Services (slides)](http://connect.linaro.org/resource/hkg18/hkg18-402/)
```
Comment (etienne-lms) [inline documentation/secure_storage.md]: I think this reference can be misleading to secure storage information. The "Secure Key Services" is not a secure storage implementation, rather a generic service that is based (for some parts) on OP-TEE secure storage solutions.

Comment (jforissier) [reply documentation/secure_storage.md]: @etienne-lms I agree. I overlooked the actual content and just checked the URL ;O

**Comment by gitfineon** [comment]
Sorry, did this on another notebook using the Web-IDE ... but there's not way to add ack and squash commits. Should be fine now.

**Comment by jforissier** [comment]
> Should be fine now.

It is, thank you.

**Pull request merged**